### ISSUE TITLE
[Bug]: Arrow KeyUp | Arrow KeyDown | PageUp | PageDown is not working properly when virtualization is enabled

### ISSUE THREAD

**Issue by dhruvindudhat** [Megabit/Blazorise#5197]
### Blazorise Version

1.3.1

### What Blazorise provider are you running on?

None

### Link to minimal reproduction, or a simple code snippet

[VirtualizeDemo.zip](https://github.com/Megabit/Blazorise/files/13724897/VirtualizeDemo.zip)

### Steps to reproduce

1. Please run the provided solution which has a listing of data in datagird with virtualization (Also Navigable flag is set to true for Windows similar functionality in datagrid)
2. Click on the first record (or any record).
3. Press ArrowDown/Up Key for multiple times.

### What is expected?

ArrowDown/Up or PageDown/Up Key should work until the last/first record of datagrid.

### What is actually happening?

Looks like due to virtualization the list of the items are changed and due to that focus on a row is not properly working and causing strange behavior of datagrid.
1.Sometimes after some selection using the ArrowDown/Up key, only scrolling happens when a selected row is not shown.
2.PageDown/Up Is just scrolling the data grid with no selected item[Should show selected item].

### Any additional comments?

In summary, When virtualization is enabled with the Navigable feature then Navigable features are not working properly.
